train.py

```diff
@@ -54,7 +53,7 @@
 target_transform = transforms.Compose(
     [
         transforms.Resize(resize_res),
-        transforms.ToTensor(),
+        transforms.PILToTensor(),
```

Comment:
En el commit creo que debería explicar cual era el error específicamente, porqué no había ningún Issues así que no sé a cual error se refiere.

Reply:
Pues el commit creo que se llama "fix scaling bug in labels", osea "corrigiendo bug de escalado en las etiquetas", básicamente es porque la función anterior escalaba los valores pero pues para las etiquetas no se quiere que se modifiquen, entonces ese es el bug. Supongo que podria explicar mejor esto en el commmit, pero tocaria hacer force-push y creo que eso no es muy bueno.

Reply:
Ya esta actualizado la descripción de la PR